modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleEmptyStateFrontendDataSet.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrontendDataSet} from '@liferay/frontend-data-set-web';
+import React from 'react';
+
+const SampleEmptyStateFrontendDataSet = ({...otherProps}) => {
+	return (
+		<FrontendDataSet
+			creationMenu={{
+				primaryItems: [
+					{
+						label: Liferay.Language.get('new-data-set'),
+						onClick: () => {
+							alert('New Data Set Modal');
+						},
+					},
+				],
+			}}
+			emptyState={{
+				description: Liferay.Language.get(
+					'start-creating-one-to-show-your-data'
+				),
+				image: '/states/empty_state.gif',
+				title: Liferay.Language.get('no-data-sets-created'),
+			}}
```

Comment:
The main goal of this task is to add empty state to tags API. Adding to a sample is a way to immediately check that new API. We don't need a sample of React API for empty state, we already have that tested in DSM.

If you wish, you can split this into two subtasks, first being just to create a placeholder tab in sample module. But, in that case, it should not be a 'fork' of React version of FDS. It should be a fork of Minimum FDS, something like:
```jsp
<frontend-data-set:headless-display
	apiURL="<%= emptyFDSDisplayContext.getAPIURL() %>"
    emptyState='<%=
		HashMapBuilder.<String, String>put(
			"description", "foo"
		).put(
			"image", "bar"
		).put(
			"title", "baz"
		).build()
	%>'
	id="<%= FDSSampleFDSNames.EMPTY %>"
/>
```

Notes on above:
- You can get URL to return an empty set by appending some impossible filter, like `?filter=('color' eq 'foo')`
- We are showing an empty FDS, so we can brand everything `Empty` (tab, FDS name, display context etc). There is no need to call everything `EmptyState`

Reply:
I' going to try/take-a-look, thanks!

Reply:
I was mislead by the title of the ticket https://liferay.atlassian.net/browse/LPS-189054